src/components/common/ModalTutorial.vue

```diff
@@ -41,7 +41,7 @@
                 {{ item }}
               </span>
               <span
-                v-if="index === 0"
+                v-if="step.title === `How to get tokens`"
```

Comment:
🤮 
What happens if Jordan changes the title of the get tokens step?

Reply:
😿